org.team_hermes.WireShare.yaml

```diff
@@ -0,0 +1,32 @@
+id: org.team_hermes.WireShare
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+command: /app/WireShare.sh
+modules:
+  - name: wireshare
+    buildsystem: simple
+    sources:
+      - type: git
+        url: https://nmatavka@bitbucket.org/nmatavka/hermes-wireshare-flatpak.git
```

Comment:
This should either be in tree or a clearly official source.

Reply:
Yeah, can it be moved under `team-hermes` bitbucket org?